test/check-application

```diff
@@ -295,22 +289,14 @@ class TestApplication(testlib.MachineCase):
         old_pid = m.execute("podman inspect --format '{{.State.Pid}}' swamped-crate".strip())
         b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button:contains(Restart)')
         b.click('#containers-containers tbody tr:contains("busybox:latest") + tr ul.dropdown-menu li a:contains(Force Restart)')
-        # We don't receive signals yet so restarting container will not update the state like running->stopped->running
-        # but we 'll probably see no change at all.
-        # Instead check that the PID of the container has changed.
         new_pid = m.execute("podman inspect --format '{{.State.Pid}}' swamped-crate".strip())
         self.assertNotEqual(old_pid, new_pid)
 
-        # Reload page after restarting the container since we don't implement proper monitoring and guessing the state
-        # can lead to unexpected behavior.
-        b.reload()
-        b.enter_page("/podman")
         self.filter_containers('all')
         b.wait_present("#containers-containers")
-        b.wait_present('#containers-containers tr:contains("swamped-crate")')
-        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
```

Comment:
Why was this dropped?

Reply:
The toggle is dropped because we don't use reload any more.

The wait_present line above is exactly the same check as the one wait_present bellow that was added, but indeed this change is not any actual change and it's confusing as part of this commit.

Reply:
Oh, of course! Thanks